packages/dashboard-provider/manual-test/manual-test.ts

```diff
@@ -0,0 +1,165 @@
+import { promisify } from "util";
+import { DashboardProvider } from "../lib";
+
+const run = async () => {
+  const provider = new DashboardProvider();
+  const send = promisify(provider.send.bind(provider));
+
+  console.log("eth_accounts");
+
+  let result = await send({
+    jsonrpc: "2.0",
+    method: "eth_accounts",
+    params: [],
+    id: 1
+  });
+
+  const [address] = result?.result;
+
+  console.log(result);
+  console.log("eth_sendTransaction");
+
+  const transaction = {
+    from: address,
+    to: address,
+    value: "16345785D8A0000" // 0.1 ETH
+  };
+
+  try {
+    result = await send({
+      jsonrpc: "2.0",
+      method: "eth_sendTransaction",
+      params: [transaction],
+      id: 1
+    });
+
+    console.log(result);
+  } catch (err) {
+    console.error(err);
+  }
+
+  console.log("eth_signTypedData");
+
+  const msgParams = [
+    {
+      type: "string", // Any valid solidity type
+      name: "Message", // Any string label you want
+      value: "Hi, Alice!" // The value to sign
+    },
+    {
+      type: "uint32",
+      name: "A number",
+      value: "1337"
+    }
+  ];
+
+  try {
+    result = await send({
+      jsonrpc: "2.0",
+      method: "eth_signTypedData",
+      params: [msgParams, address],
+      id: 1
+    });
+
+    console.log(result);
+  } catch (err) {
+    console.error(err);
+  }
+
+  console.log("eth_signTypedData_v4");
+
+  const chainIdResult = await send({
+    jsonrpc: "2.0",
+    method: "net_version",
+    params: [],
+    id: 1
+  });
+
+  const chainId = chainIdResult?.result ?? 1;
+
+  // Code taken from https://docs.metamask.io/guide/signing-data.html#sign-typed-data-v4
+
+  const msgParams_v4 = JSON.stringify({
```

Comment:
cc @kevinweaver @kevinbluer @MicaiahReid I entrust with you three the knowledge that there exists this example of doing EIP-712 inside the Truffle tests. 

(okay, and anyone else that sees this... let's remember that we have this example of how to do EIP-712! like, we may want to reference this later!)